lib/workout.js

```diff
@@ -78,6 +95,17 @@ module.exports = class Workout extends Content {
       markdown += '\n';
     }
 
+    // Append exercises
+    if (this.exercises != null && this.exercises.length > 0) {
+      for (let exercise of this.exercises) {
+        markdown += "---\n## Exercise\n";
+        markdown += `${exercise['link-type']}-link: ${exercise['link']}\n`;
+        markdown += `answer: ${exercise.answer.toString().replace(/\t/g, "\n\t")}\n`;
```

Comment:
What if there is unhandled metadata in the yaml for an exercise? Make sure to loop over and append other metadata.